src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -347,6 +358,24 @@ def convert_yaml_to_test(data):
                 new_body["passFailCriteria"]["passFailMetrics"][metric_id][
                     "requestName"
                 ] = name
+    if data.get("autoStop") is not None:
+        if (isinstance(data["autoStop"], str)):
```

Comment:
Should we check if this value is set to disable

Reply:
we can do that, if it's not disable, do we want to throw an error, @sulabh-msft?

Reply:
will throw InvalidArgumentValueError